site/en/docs/webstore/best_practices/index.md

```diff
@@ -207,9 +200,5 @@ change; if that happens, your web app might be recategorized.
 [1]: /docs/apps
 [2]: /docs/webstore/launching#pre-launch-checklist
```

Comment:
/docs/webstore/launching is being removed in this PR.

Reply:
Removed the reference to this removed page.